com.unity.perception/Tests/Runtime/GroundTruthTests/LabelEntryMatchCacheTests.cs

```diff
@@ -131,5 +131,43 @@ public IEnumerator TryGet_ReturnsFalse_ForNonMatchingLabel_WhenAllObjectsAreDest
             }
         }
 
+        [UnityTest]
+        public IEnumerator TryGet_ReturnsFalse_ForMatchingLabelWithDisabledLabelingComponent()
```

Comment:
We should have a test for the keypoint labeler as well, since it does not use any of the tested codepaths.